src/components/meeting/meetingTime/JitsiComponentWrapper.js

```diff
@@ -18,18 +18,25 @@ import CloseIcon from '@material-ui/icons/Close';
 import Swal from "sweetalert2";
 import {MultipleTasksWrapper} from "../../tasks/MultipleTasksWrapper";
 
-export const JitsiWrapper = ({ admin, roomName, displayName, interviewId, companyId }) => {
+export const JitsiWrapper = ({ admin, roomName, displayName, interviewId, companyId, interviewKind }) => {
 
     const [jitsiHeight, setJitsiHeight] = useState('calc(100vh - 60px)')
     const [jitsiWidth, setJitsiWidth] = useState('100%')
     const [actionComponent, setActionComponent] = useState(null)
     const [taskStageUUID, setTaskStageUUID] = useState(null)
+    const [shouldShow, setShouldShow] = useState([])
 
     useEffect(() => {
-        interviewAPI.getTaskStageUUIDByInterviewId(interviewId)
-            .then(data => setTaskStageUUID(data.taskStageUUID))
+        if (interviewKind === 'technical') {
+            interviewAPI.getTaskStageUUIDByInterviewId(interviewId)
+                .then(data => setTaskStageUUID(data.taskStageUUID))
+        }
     }, [interviewId, companyId])
 
+    useEffect(() => {
+        setShouldShow(interviewKind === "hr" ? ['whiteboard', 'default'] : interviewKind === 'technical' ? ['whiteboard', 'default', 'task'] : [])
+    }, [interviewKind, interviewId, companyId])
+
```

Comment:
Może głupie pytanie jak działają dwa useEffecty obok siebie?

Reply:
Normalnie, nie wchodzą sobie w drogę. To tak jakbyś na starcie aplikacji dwie funkcje wywołał.